x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -367,6 +367,16 @@ export function getDeepLinks(
           deepLinks: [],
         };
       }
+      if (
```

Comment:
Super nit: could you add a comment in here just noting that this logic applies to both the main navigation and side navigation?

Reply:
Added the comment at the function header